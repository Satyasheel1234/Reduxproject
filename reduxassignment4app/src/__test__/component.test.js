import React from 'react';
import Listview from '../component/ListView';
import { create} from 'react-test-renderer';

describe('Snapshot component',() => {
    test('testing header component' ,() => {
        let tree = create(<Listview/>)
        expect(tree.toJSON()).toMatchSnapshot();
    })
    
    
});