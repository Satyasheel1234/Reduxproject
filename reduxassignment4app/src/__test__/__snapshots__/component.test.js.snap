// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot component testing header component 1`] = `<div />`;
